client/src/store/new-request-module.ts

```diff
@@ -1995,9 +1988,10 @@ export class NewRequestModule extends VuexModule {
   @Action
   async addRequestActionComment (data) {
     try {
-      let requestAction = this.requestActionOriginal || this.request_action_cd
-      let { shortDesc } = this.requestActions.find(request => request.value === requestAction)
-      let msg = `*** ${shortDesc} ***`
+      const requestAction = this.requestActionOriginal || this.request_action_cd
+      const action = this.requestActions.find(request => request.value === requestAction)
+      const { shortDesc } = action || { shortDesc: 'action not found' }
+      const msg = `*** ${shortDesc} ***`
```

Comment:
Nice, good fix. 
Curious though, where this would be blank or fail in the past, does this default msg have any implications downstream? 

To be honest, i don't really know what the message is used for, though i do know the `additional info` field is additive to this msg.

Reply:
It might have been blank due to other issues (eg, app state reset before unawaited async actions completed).

This just forms part of an error message, so should be no impact.

Reply:
Totally! I can't think of a downside. It's 100% an improvement

Reply:
Ya. Before it was causing an exception, so the rest of this method wasn't being executed.